hazelcast/src/main/resources/hazelcast-config-4.0.xsd

```diff
@@ -2525,12 +2525,23 @@
         </xs:simpleContent>
     </xs:complexType>
     <xs:complexType name="index">
-        <xs:simpleContent>
-            <xs:extension base="xs:string">
-                <xs:attribute name="ordered" type="xs:boolean" use="optional" default="false"/>
-            </xs:extension>
-        </xs:simpleContent>
+        <xs:all>
+            <xs:element name="attributes" type="index-attributes"/>
+        </xs:all>
+        <xs:attribute name="name"/>
+        <xs:attribute name="type" type="index-type" default="SORTED"/>
     </xs:complexType>
+    <xs:complexType name="index-attributes">
+        <xs:sequence>
+            <xs:element name="attribute" type="xs:string" maxOccurs="unbounded"/>
```

Comment:
255?

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/15537/commits/2163cbdd5ac5290a80317aa92679e8ded7520b42